scripts/createReleasePR.mjs

```diff
@@ -198,11 +198,73 @@ async function findForkRemote() {
   }
 }
 
+/**
+ * Find the latest major version from the upstream remote
+ * @param {string} remote - The name of the remote (e.g., 'upstream')
+ * @returns {Promise<string>} The latest major version (e.g., '7', '6', etc.)
+ */
+async function findLatestMajorVersion() {
+  try {
+    // fetch tags from the GitHub API and return the last one
+    const { data: tags } = await octokit.rest.repos.listTags({
+      owner: ORG,
+      repo: REPO,
+    });
```

Comment:
In which order does this return the tags? 

I'm a bit worried that if the v7 tag has been created after v8, then we'll read the latest major as v7 rather than v8.

Reply:
Ah, nice 👍